ui/src/components/table/TableHeader.jsx

```diff
@@ -71,13 +73,23 @@ function TableHeader({
         );
     };
 
+    const getInputCountStatus = () => {
+        const enabledRowCount = allFilteredData.filter((item) => !item.disabled).length;
+        const showCountstatus = `(${enabledRowCount} of ${totalElement} enabled)`;
+
+        return (
+            <Typography as="span">
+                {totalElement}
+                {totalElement > 1 ? _(` ${itemLabel}s`) : _(` ${itemLabel}`)}
+                &nbsp; {totalElement >= pageSize && showCountstatus}
+            </Typography>
+        );
+    };
+
     return (
         <TableHeaderWrapper>
             <div>
-                <span className="inputNumber">
```

Comment:
I also wondered where it is used. So, it is used in smartx
https://github.com/splunk/addon-factory-smartx-ui-test-library/blob/07fe4b9ebc9df5f02d17b321227d723e0884d950/pytest_splunk_addon_ui_smartx/components/table.py#L85